### PR TITLE
feat(sql): Refactor delete and update handler

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Move delete and update to CRUD namespace

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b refactor-sql-deletehandler-and-updatehandler -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by ChrOertlin — APPROVED** [review]
💯  Nice!

File: cg/store/crud/delete.py [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+from cg.store.api.base import BaseHandler
+from cg.store.filters.status_flow_cell_filters import (
+    FlowCellFilter,
+    apply_flow_cell_filter,
+)
+from cg.store.models import Case, Flowcell, Sample, SampleLaneSequencingMetrics
+
+
+class DeleteDataHandler(BaseHandler):
+    """Contains methods to delete business data model instances"""
```
Comment (diitaz93) [inline cg/store/crud/delete.py:15]:
```suggestion
    """Contains methods to delete business data model instances."""
```

**Review by diitaz93 — APPROVED** [review]
👍

**Comment by henrikstranneheim** [comment]
cg get case justhusky 👍 
```
|----------------+-----------------+------------+---------------+--------------------+------------+-------------|
| hugelymodelbat | ACC5339A4--300M | cust000    | WGXCUSC000    | Ordered 2019-05-08 | priority   | Yes         |
+----------------+-----------------+------------+---------------+--------------------+------------+-------------+
+---------------------+------------+-----------+
| Analysis Date       | Pipeline   | Version   |
|---------------------+------------+-----------|
| 2023-12-07 10:32:30 | mip-dna    | 12.0.3    |
+---------------------+------------+-----------+
```

**Pull request merged**